presto-docs/src/main/sphinx/release/release-0.212.rst

```diff
@@ -0,0 +1,29 @@
+=============
+Release 0.212
+=============
+
+General Changes
+---------------
+
+* Add the :func:`ST_LineString` function to form a ``LineString`` from an array of points.
+* Enhanced :func:`ST_ConvexHull` and :func:`convex_hull_agg` functions to support geometry collections.
+* Improve performance for some query plans that include ``MarkDistinct`` or ``DistinctLimit``.
```

Comment:
Rather than discussing internal plan node names, please explain it in terms of SQL that a user would write.